sdk/go/client_test.go

```diff
@@ -0,0 +1,97 @@
+package feast
+
+import (
+	"context"
+	"testing"
+
+	"github.com/feast-dev/feast/sdk/go/mocks"
+	"github.com/feast-dev/feast/sdk/go/protos/feast/serving"
+	"github.com/feast-dev/feast/sdk/go/protos/feast/types"
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-cmp/cmp"
+	"github.com/opentracing/opentracing-go"
+)
+
+func TestGetOnlineFeatures(t *testing.T) {
+	tt := []struct {
+		name    string
+		req     OnlineFeaturesRequest
+		recieve OnlineFeaturesResponse
+		want    OnlineFeaturesResponse
+		wantErr bool
+		err     error
+	}{
+		{
+			name: "valid",
```

Comment:
This name can be more informative if you're testing for more than just a "valid" request

Reply:
Corrected.